crates/cxx-qt-lib-headers/include/core/qbytearray.h

```diff
@@ -63,5 +63,7 @@ qbytearrayToUpper(const QByteArray& byteArray);
 QByteArray
 qbytearrayTrimmed(const QByteArray& byteArray);
 
```

Comment:
remove the empty line here i think, as these were grouped into methods that are just Qt wrappers and ones that are helpers to Rust

Reply:
also keep the order A->Z so this should be after Len and before Remove